Cooper/ClientApp/src/app/models/user.ts

```diff
@@ -0,0 +1,13 @@
+export class User {
+    id: number;
+    nickname: string;
+    email: string;
+    photoURL: string;
+
+    // IsVerified: boolean;
+    // IsCreator: boolean;
+    // IsBanned: boolean
+
+    platformLanguage: string;
+    platformTheme: string;
```

Comment:
looks like enums and separate class 

Question: do we have separate tables for languages and theme in database?

Reply:
Nope. This feature haven't been disscussed properly yet